fedora-infra/fasjson
# Rest API POC

Related issues:
* #7 
* #13

This is a poc which proposes a new basic structure, it is not 100% ready as I need to link the LDAP side of things in flask

I am using a feature branch until the full implementation is ready, then we can merge it back in master.

## Verification steps

* Run tests: `make test`
* Initialize the development VM environment: `vagrant up --no-parallel`
* SSH into the fasjson guest machine: `vagrant ssh fasjson`
* Run `kinit admin@EXAMPLE.NET` and input the following password: `adminPassw0rd!`
* Run `curl --negotiate -u : http://fasjson.example.test/fasjson/v1/groups`

The final output should be something like this:

```
{"error": {"message": "method not implemented", "data": {"reason": "method not implemented"}}}
```
